libs/profile/client/src/lib/profile.client.ts

```diff
@@ -85,9 +83,9 @@ export class ProfileClient {
         {},
         'delete'
       );
-    } catch (err) {
-      this.log.error('profile.deleteCache.failed', uid, err);
-      throw err;
+    } catch (error) {
+      this.log.error(error);
```

Comment:
nit. ideally this would be something like the code below, so that the additional context of the `uid` isn't lost. This can be fixed up at a later stage though.

```
const deleteCacheError = new ProfileClientDeleteCacheError('Error occurred during delete cache', uid, error)
this.log.error(deleteCacheError);
throw deleteCacheError;
```